### DISCUSSION TITLE
Stelllarium won't connect to scope in Linux

### DISCUSSION THREAD

**Discussion by GaryN827** [Stellarium/stellarium#2256] [Q&A]
I recently installed Stellarium 21.3 on a computer with Linux Ubuntu 20.4. My mount is an Orion EQG which is a Synscan compatible with the Skywatcher mount.   I set up the plugin for telescope control to direct serial control, the Sky Watcher Synscan Version 3 or later selection and the serial port to /dev/ttyUSB0.  I am using a Prolific USB to serial converter between the computer and the telescope serial port.  The problem I am having is that when I go back and push the start button nothing happens.  
I have myself setup as a user in Dialout which I confirmed with the id -Gn command.  
gary adm dialout cdrom sudo dip plugdev lpadmin lxd sambashare
I confirmed ttyUSB0 is part of dialout by the ls -l /dev/ttyUSB0 command.  
crw-rw---- 1 root dialout 188, 0 Feb  8 15:27 /dev/ttyUSB0
Using command sudo dmesg shows that Ubuntu does recognize the converter with its driver.
[   20.059656] usbcore: registered new interface driver pl2303
[   20.059665] usbserial: USB Serial support registered for pl2303
[   20.059678] pl2303 3-2:1.0: pl2303 converter detected
[   20.061015] usb 3-2: pl2303 converter now attached to ttyUSB0
And lsusb command shows the prolific converter connected
Bus 003 Device 002: ID 067b:2303 Prolific Technology, Inc. PL2303 Serial Port
I can connect the scope with Stellarium on Windows and it controls with no problems.  So is there something I am missing in the Ubuntu or Stellarium setup?  Thanks

**Comment by RJ3** [comment]
Was this ever solved?
Having same issue with Stellarium 25.1. Works perfect on Windows but can't connect on Linux.

**Reply by 10110111** [reply]
If you post Stellarium log file, we might be able to diagnose the problem.

**Reply by RJ3** [reply]
I don't know what I changed but it's connecting now so we can close this. If it fails again I will attach a log!
